lib/openhab/dsl/rules/builder.rb

```diff
@@ -380,6 +380,30 @@ class BuilderDSL
         #
         prop :enabled
 
+        #
+        # Return all {Item Items} (or {GroupItem::Members GroupItem::Members}) referenced
+        # by the specified triggers types in this rule.
+        #
+        # @param [Symbol, Array<Symbol>] trigger_types Trigger types to search for dependencies
```

Comment:
* Is there a list of trigger types the user can refer to?
* Why does the default not include `:received_command`?

Reply:
they're named the same as the trigger methods.

because for the most common use case for this (`dependencies.all?(&:state?)`) you probably don't care if the item receiving the command has a state.

ooh, I should uniq the final result though